input/sdl_gamepad.c

```diff
@@ -17,15 +17,20 @@
 
 #include <SDL.h>
 #include <stdbool.h>
+#include <pthread.h>
 #include "common/common.h"
 #include "osdep/atomic.h"
 #include "common/msg.h"
 #include "input.h"
 #include "input/keycodes.h"
 
-struct priv {
-    Uint32 cancel_wakeup;
-};
+Uint32 gamepad_cancel_wakeup;
```

Comment:
Forgot static?

Reply:
yes, I can't see warnings from all the clang spam